src/model/migrations.js

```diff
@@ -481,6 +487,15 @@ export default async function runMigrations() {
         logger.log(JSON.stringify({ pinnedCoinsMigrated }, null, 2));
         logger.log(JSON.stringify({ hiddenCoinsMigrated }, null, 2));
 
+        mmkv.set(
```

Comment:
Our protocol is to not change existing migrations once they have already been created - everything should be handled in the new migration you are creating